app/models/author.rb

```diff
@@ -1,3 +1,14 @@
+# == Schema Information
```

Comment:
Let's aggressively delete these. They go out of date too easily and it's just as easy to pop open schema.rb for the source of truth.